.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: weekly
```

Comment:
We'll scan go dependencies by Trivy.

Reply:
What do you mean by scanning, does it includes upgrades ?

Reply:
```suggestion
```